packages/gatsby-theme-carbon/src/components/markdown/markdown.module.scss

```diff
@@ -0,0 +1,20 @@
+@import '~carbon-components/scss/globals/scss/vars';
+@import '~@carbon/elements/scss/type/type';
+
+.paragraph {
+  @include carbon--type-style('body-long-02');
+  margin-bottom: $spacing-04;
+}
+
+// Responsive by default
+.responsive {
+  // 7 columns wide
+  @include carbon--breakpoint('md') {
+    width: 62.5%;
+  }
+
+  // 5 columns wide
+  @include carbon--breakpoint('lg') {
+    width: 58.333%;
```

Comment:
```suggestion
    width: 66.6667%;
```

Reply:
should be 8 per https://github.com/carbon-design-system/gatsby-theme-carbon/issues/55